src/storage/src/hummock/store/version.rs

```diff
@@ -67,6 +68,7 @@ impl StagingVersion {
         &'a self,
         epoch: HummockEpoch,
         compaction_group_id: CompactionGroupId,
+        table_id: TableId,
```

Comment:
We can remove passing the compaction_group_id and only do filtering by table id.

Reply:
Does `imm` only belong to one compaction-group?

Reply:
Yes. But imm also belongs to only one table. Therefore we don't have to store the compaction group id in `imm`. table id is enough for filtering. If we don't have to pass compaction group id when we do filtering, we can remove the dependency on compaction group client on CN.

Reply:
> We can remove passing the compaction_group_id and only do filtering by table id.

fixed